src/utils/download_file.test.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { getTempFolder } from './temp_folder';
+import { downloadFile } from './download_file';
+import * as fs from 'fs';
+
+describe('downloadFile function', () => {
+	const validDownloadLink = 'https://arweave.net/pVoSqZgJUCiNw7oS6CtlVEd8gREQlpRbccrsMLkeIuQ';
+	const invalidDownloadLink = 'https://arweave.net/pVoSqZgJUCiNw7oS6CtlVEV8gREQlpRbccrsMLkeIuQ';
+	const destinationFileName = 'cat.jpg';
+	const tempFolderPath = getTempFolder();
```

Comment:
I would expect this folder to be cleaned up in `after(() => { cleanUpTempFolder() })`